src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionUIService.cs

```diff
@@ -33,34 +31,34 @@ public void Dispose()
     {
         if (_serviceProvider is not null)
         {
-            IDesignerHost host = (IDesignerHost)_serviceProvider.GetService(typeof(IDesignerHost));
+            IDesignerHost? host = (IDesignerHost?)_serviceProvider.GetService(typeof(IDesignerHost));
```

Comment:
Let's use T version of this method
```suggestion
            IDesignerHost? host = _serviceProvider.GetService<IDesignerHost>();
```